optaplanner-core/src/main/java/org/optaplanner/core/config/heuristic/selector/move/composite/CartesianProductMoveSelectorConfig.java

```diff
@@ -65,18 +66,40 @@
 
     private Boolean ignoreEmptyChildIterators = null;
 
+    // ************************************************************************
+    // Constructors and simple getters/setters
+    // ************************************************************************
+
     public CartesianProductMoveSelectorConfig() {
     }
 
     public CartesianProductMoveSelectorConfig(List<MoveSelectorConfig> moveSelectorConfigList) {
         this.moveSelectorConfigList = moveSelectorConfigList;
     }
 
+    /**
+     * @deprecated in favor of {@link #getMoveSelectorList()}.
+     * @return sometimes null
+     */
+    @Deprecated
     public List<MoveSelectorConfig> getMoveSelectorConfigList() {
```

Comment:
See `SolverConfig.setPhaseList(List<PhaseConfigList)`
The naming convention is to drop the "Config" in the name (but not in the typing).